templates/docs/examples/patterns/forms/fieldset-required.html

```diff
@@ -0,0 +1,24 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Forms / Fieldset required{% endblock %}
+
+{% block standalone_css %}patterns_forms{% endblock %}
+
+{% block content %}
+<form>
+  <fieldset>
+    <legend class="is-required" for="list-input-1">Required input</legend>
```

Comment:
I don't think the `for` attribute is supported for the `<legend>` element ([MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend))